File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/Java/Python/s826491904_commented.py
n = int(input())
max_v = min_v = int(input())
for _ in range(n-1):
    num = int(input())
    max_v = max(max_v, num)
    min_v = min(min_v, num)
print(max_v - min_v)  # End of Code

File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/direct/comment/comment/direct/Java/Go/s826491904_commented.go
package main

import (
	"fmt"
)

func main() {
	var N, max, min int
	fmt.Scan(&N)
	fmt.Scan(&max)
	min = max
	for i := 1; i < N; i++ {
		var in int
		fmt.Scan(&in)
		if in > max {
			max = in
		}
		if in < min {
			min = in
		}
	}
	fmt.Println(max - min)
}

// End of Code
